addons/vrm/import_vrm.gd

```diff
@@ -801,7 +801,7 @@ func _import_scene(path: String, flags: int, bake_fps: int):
 		gstate = load("res://addons/godot_gltf/GLTFState.gdns").new()
 		gltf = load("res://addons/godot_gltf/PackedSceneGLTF.gdns").new()
 	print(path);
-	var root_node : Node = gltf.import_gltf_scene(tmp_path, 0, 1000.0, gstate)
+	var root_node : Node = gltf.import_gltf_scene(tmp_path, 0, 1000.0, 2194432, gstate)
```

Comment:
Compression flag?

Reply:
Please see the code here:
https://github.com/V-Sekai/godot-blender/blob/godot3/addons/blend/import_blend.gd#L57..L61

```gd
	var import_config_file = ConfigFile.new()
	import_config_file.load(path + ".import")
	var compression_flags: int = import_config_file.get_value("params", "meshes/compress", 0)
	# ARRAY_COMPRESS_BASE = (ARRAY_INDEX + 1)
	compression_flags = compression_flags << (VisualServer.ARRAY_INDEX + 1)
	if import_config_file.get_value("params", "meshes/octahedral_compression", false):
		compression_flags |= VisualServer.ARRAY_FLAG_USE_OCTAHEDRAL_COMPRESSION
```

then, use `compression_flags` instead of `2194432` as the argument.

Additionally, you must move this `var root_node : Node` into the if statement. In the GDNative case, that argument is not supported.

I forgot how to suggest changes but do this:
```gd
	var root_node : Node
	if type_exists("GLTFState") and type_exists("PackedSceneGLTF"):
		print("VRM: Using builtin gltf module")
		# if ClassDB.can_instance("GLTFState") and ClassDB.can_instance("PackedSceneGLTF"):
		gstate = ClassDB.instance("GLTFState")
		gltf = ClassDB.instance("PackedSceneGLTF")
		root_node = gltf.import_gltf_scene(tmp_path, 0, 1000.0, compression_flags, gstate)
	else:
		print("VRM: Using GDNative godot_gltf")
		gstate = load("res://addons/godot_gltf/GLTFState.gdns").new()
		gltf = load("res://addons/godot_gltf/PackedSceneGLTF.gdns").new()
		root_node = gltf.import_gltf_scene(tmp_path, 0, 1000.0, gstate)
```